test/provision/k8s_install.sh

```diff
@@ -23,7 +23,6 @@ export KUBEADM_CRI_SOCKET="/var/run/dockershim.sock"
 export KUBEADM_SLAVE_OPTIONS=""
 export KUBEADM_OPTIONS=""
 export K8S_FULL_VERSION=""
-export INSTALL_KUBEDNS=1
 export DNS_DEPLOYMENT="${PROVISIONSRC}/manifest/dns_deployment.yaml"
```

Comment:
When is this file created?

Reply:
@aanm depending upon the K8s version being used a symlink is created accordingly to kube-dns or core-dns.